Sky Atlas/Sky Atlas.html

```diff
@@ -24712,39 +24759,69 @@ <h1>Agent Scope Database</h1>
         </tr>
         <tr>
           <td>
-            <dfn>Root Edit Proposal Submission Requirements Exception</dfn>
+            <dfn>Root Edit Proposal Submission Requirements Exception For Nested Contributors</dfn>
+          </td>
+          <td>Spark</td>
+          <td>Core</td>
+          <td><p>Nested Contributors are always authorized to submit Artifact Edit Proposals and do not have to fulfill the token-holding requirements defined in Root Edit Proposal Submission. However, all other procedural requirements within the Root Edit process continue to apply.</p><p>To see the Agent’s Nested Contributors, see <dfn>Phoenix Labs</dfn>.</p></td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>Root Edit Proposal Submission Requirements Exception For Spark Foundation</dfn>
+          </td>
+          <td>Spark</td>
+          <td>Core</td>
+          <td><p>While it is a core service provider to Spark, the Spark Foundation is authorized to submit Artifact Edit Proposals and does not have to fulfill the token-holding requirements defined in Root Edit Proposal Submission. However, all other procedural requirements within the Root Edit process continue to apply.</p></td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>Short-Term Transitionary Measures</dfn>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td>For proposals that solely entail a buyback or a grant of SPK tokens, the requirement that SPK token holders must hold at least 1% of the circulating token supply to submit a proposal is waived. However, all other procedural requirements within the Root Edit process continue to apply.</td>
+          <td><p>Until the Powerhouse system supports submitting Artifact Edit Proposals, SPK token holders may submit Artifact Edit Proposals by posting them to the Sky Forum under the “Spark” category. The title of the post must include the text “Spark Artifact Edit Proposal”. The post must include cryptographic proof that the author controls an account holding the required percentage of the total SPK token supply specified in Root Edit Proposal Submission.</p></td>
         </tr>
         <tr>
           <td>
             <dfn>Root Edit Expert Advisor Review</dfn>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td>A future iteration of the Spark Artifact will specify guidelines for obtaining specialized review of proposals requiring advanced technical or financial analysis.</td>
+          <td><p>A future iteration of the Spark Artifact will specify guidelines for obtaining specialized review of proposals requiring advanced technical or financial analysis.</p></td>
```

Comment:
nit: You don't need p tags for a single line entry

```suggestion
          <td>A future iteration of the Spark Artifact will specify guidelines for obtaining specialized review of proposals requiring advanced technical or financial analysis.</td>
```